test/test_pub_sub.py.in

```diff
@@ -222,17 +242,23 @@ class PerformanceTestResults(unittest.TestCase):
                                          performance_log_prefix_cpumem_sub,
                                          performance_log_prefix_pub,
                                          performance_log_prefix_sub,
-                                         proc_info):
-        # self.addCleanup(_cleanUpLogs, performance_log_prefix_cpumem_pub)
```

Comment:
These directories are getting created above by `mkstemp`. Why not delete them when the test is completed?

Reply:
removed at the end of the file https://github.com/ros2/buildfarm_perf_tests/pull/15/commits/97b06a36ebe42b795f3e36d2cf602058ec0a4b57